external/fv3fit/tests/training/test_graph.py

```diff
@@ -115,3 +118,14 @@ def sample_func():
         )
 
     return sample_func
+
+
+def test_graph_builder():
+    graph = build_graph(2)
+    edges = list(zip(graph[0], graph[1]))
+    assert (0, 1) in edges
```

Comment:
Add a comment after each assertion which connection it is testing or add intermediate variables with the name.

Reply:
Added comments.